.mailmap

```diff
@@ -7,3 +7,5 @@ Bill Mills <mills.wj@gmail.com> <mills.wj@gmail.com>
 Ivan Gonzalez <iglpdc@gmail.com> <iglpdc@users.noreply.github.com>
 Pauline Barmby <pbarmby@uwo.ca> <pbarmby@uwo.ca>
 Raniere Silva <raniere@ime.unicamp.br>
+Daisie Huang <daisieh@gmail.com>
+Emmanouil Farsarakis <farsarakis@gmail.com> efarsarakis <farsarakis@epcc.ed.ac.uk>
```

Comment:
I usually don't bother with the `Commit Name` portion of the entry, since it's unlikely that you'll have `farsarakis@epcc…` authors besides Emmanouil.  So probably just:

    Emmanouil Farsarakis <farsarakis@gmail.com> <farsarakis@epcc.ed.ac.uk>